Microsoft.Azure.Cosmos/src/Fluent/CosmosClientBuilder.cs

```diff
@@ -424,12 +424,51 @@ public CosmosClientBuilder WithConsistencyLevel(Cosmos.ConsistencyLevel consiste
         }
 
         /// <summary>
-        /// If Open Telemetry listener is subscribed for Azure.Cosmos namespace, There are <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/> you can leverage to control it.<br></br>
+        /// Distributed Tracing allows to capture timing data for specific operations and send it to a data store for viewing.
+        /// <para>
+        /// Usage with AppInsights: AppInsight SDK support it out of the box. ref. <seealso href="https://github.com/Microsoft/ApplicationInsights-dotnet/releases" />. You can directly start using it with AppInsight SDK.
+        /// </para>
+        /// <para>
+        /// Usage with Opentelemetry or Custom Listener: Subscribe for "Azure.Cosmos" source. Then, <see cref="System.Diagnostics.Activity"/> is going to generate for the SDK operations along with <see cref="System.Diagnostics.Tracing.EventSource"/> with Request Diagnostic log information (based on <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/>).
+        /// SDK don't correlate <see cref="System.Diagnostics.Activity"/> and <see cref="System.Diagnostics.Tracing.EventSource"/>, it has to be taken care at custom listener by the application.
+        /// </para>
+        /// Using <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/> you can control the <see cref="System.Diagnostics.Tracing.EventSource"/> traces containing request diagnostics logs.
         /// </summary>
+        /// <example>
+        /// Enable distributing tracing with default configuration
+        /// <code language="c#">
+        /// <![CDATA[ 
+        ///  CosmosClientBuilder cosmosClientBuilder = new CosmosClientBuilder(accountEndpoint: endpoint, authKeyOrResourceToken: key);
+        ///  CosmosClient cosmosClient = cosmosClientBuilder
+        ///                                 .WithDistributingTracing()
+        ///                                 .Build();
+        /// ]]
```

Comment:
```suggestion
        /// ]]>
```

Missing `>` which breaks the content

Reply:
This is all across the examples